### ISSUE TITLE
[A11y] During activity transitions the talkback outputs Oppia first [BLOCKED]

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#2619]
During activity transitions the talkback outputs Oppia first. Instead it should be something more meaningful.

Blocked because we need more context about how others app handle activity transitions and what should be the correct format.

**Comment by rt4914** [comment]
Duplicate #2743
